src/iticket.cpp

```diff
@@ -457,18 +459,17 @@ showTickets( char *inName ) {
     }
 }
 
-
-std::string
-makeFullPath( const char *inName ) {
-    std::stringstream fullPathStream;
-    if ( strlen( inName ) == 0 ) {
-        return std::string();
-    }
-    if ( *inName != '/' ) {
-        fullPathStream << cwd << "/";
+auto makeFullPath(const char *inName) -> std::optional<std::string> {
```

Comment:
This looks like the implementation from `istream` and other functions.

Let's just rename this function to match what's in `istream`. That way we know they are related and no one's confused.